mutacc/cli/root.py

```diff
@@ -41,20 +42,27 @@ def cli(context, loglevel, config_file, root_dir):
         with open(config_file, 'r') as in_handle:
             cli_config = yaml.load(in_handle, Loader=yaml.FullLoader)
 
-
     mutacc_config = {}
-    mutacc_config['host'] = cli_config.get('host') or 'localhost'
-    mutacc_config['port'] = cli_config.get('port') or 27017
-    mutacc_config['username'] = cli_config.get('username')
-    mutacc_config['password'] = cli_config.get('password')
-    mutacc_config['db_name'] = cli_config.get('database') or 'mutacc'
-
-    #Check the root_dir and add to mutacc_config
-    root_dir = cli_config.get('root_dir') or root_dir
-    if not root_dir:
-
-        LOG.warning('Please provide a root directory, through option --root-dir or in config_file')
-        context.abort()
+    if demo:
+        mutacc_config['host'] = 'localhost'
+        mutacc_config['port'] = 27017
+        mutacc_config['db_name'] = 'mutacc-demo'
+        root_dir = make_dir('~/mutacc_root_demo')
+
+    else:
+        mutacc_config['host'] = cli_config.get('host') or 'localhost'
+        mutacc_config['port'] = cli_config.get('port') or 27017
+        mutacc_config['username'] = cli_config.get('username')
+        mutacc_config['password'] = cli_config.get('password')
+        mutacc_config['db_name'] = cli_config.get('database') or 'mutacc'
+
+        #Check the root_dir and add to mutacc_config
+
+        root_dir = cli_config.get('root_dir') or root_dir
+        if not root_dir:
+
+            LOG.warning('Please provide a root directory, through option --root-dir or in config_file')
```

Comment:
Ok but then you have a root dir option , just provide a default to the package itself